docs/Guides/zopen.md

```diff
@@ -0,0 +1,119 @@
+# zopen framework 
+
+The following is a description of the zopen tools provided in the meta repo. To obtain the latest tools, you can clone and set up the environment as follows:
+
+```
+git clone git@github.com:ZOSOpenTools/meta.git
+cd meta
+. ./.env
+```
+
+## If you are using z/OS Open Tools
+
+### zopen init
+To initialize the zopen installation directory to a location other than the default ($HOME/zopen), you can run the command zopen init and specify the desired directory. This will configure the directory for future use. Subsequently, tools like zopen download will download and install files to this specified directory."
+
+### zopen cacert
+
+To update the cacert.pem file, you can use `zopen update-cacert`. This will download the latest cacert.pem file https://curl.se/docs/caextract.html. This cacert.pem file is then used by other tools such as `zopen download` and `zopen build`.
+
+### zopen download
+
```

Comment:
should we have a reference to zopensetup here in case they end up here first?

Reply:
Good idea, adding it